src/atoms/FeaturedSection/featuredSection.module.css

```diff
@@ -0,0 +1,26 @@
+.featuredSection {
+  padding: 20px 28px;
+}
+
+.featuredSection p {
+  margin-bottom: 1em;
```

Comment:
I removed margin from this paragraph because it was making no diference in layout, but please explain me with more details this ideia of extracting margins to a component.